### ISSUE TITLE
Internet Users can add attachments to licenses; something we don't want to happen.

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#11577]
> What application are you using?

AMANDA

> Describe the problem.

Internet Users can add attachments to licenses; something we don't want to happen.

> Describe the outcome you'd like to see when this feature is implemented.

Internet Users would no longer be able to add attachments online to their license.

> Describe any workarounds you currently have in place or alternative solutions you've considered.

Currently no work arounds.

> Requested By
Kimberly P.

[Attachment](https://api.knack.com/v1/applications/595d00ebd315cc4cb98daff4/download/asset/63fcfd69ea3e7c00291e1986/microsoftteamsimage17.png) (18.59kb)

Request ID: DTS23-106520

**Comment by Nadin-Nader** [comment]
@vish can you add this to your scoping list please?

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader  Ready for Dev Deployment , SCTASK0301103

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT Is this ready to go to TEST?

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Ready for UAT on Test

**Comment by Nadin-Nader** [comment]
Sent Teams message to Kyle and ROW to please UAT this on TEST and let me know the results. Waiting on Response

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT this has been approved by Kyle to go to PROD

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Created Prod promotion request - SCTASK0313020

**Comment by Nadin-Nader** [comment]
Still not deployed to PROD

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Changes deployed to Prod, Ready for UAT

**Comment by Nadin-Nader** [comment]
Sent Message to Kyle and ROW Team to please UAT and let us know if this is good to close or not now that it is on PROD. Waiting on Response

**Comment by Nadin-Nader** [comment]
This has been approved on PROD by ROW, closing
